app/models/manageiq/providers/redhat/infra_manager/vm/remote_console.rb

```diff
@@ -16,6 +16,10 @@ def validate_remote_console_acquire_ticket(protocol, options = {})
             "#{protocol} remote console requires the vm to be running.") if options[:check_if_running] && state != "on"
     end
 
+    def remote_viewer_console_file
+      ext_management_system.ovirt_services.remote_viewer_console_file(self)
+    end
+
```

Comment:
You might need to add a `remote_viewer_console_queue` method to request this over the queue like the acquire_ticket method below